tests/test_cli.py

```
@@ -0,0 +1,452 @@
+
+    mock_extract = MagicMock(spec=OpenStreetMapExtract)
+    mock_extract.id = "test"
+    mock_extract.name = "test"
+    mock_extract.file_name = "test"
+    mock_extract.geometry = box(0, 0, 1, 1)
+
+    mock_result = MagicMock()
+    mock_result.extracts = [mock_extract]
+    mock_result.matched_extracts = [mock_extract]
+    mock_result.sources_used = []
+    mock_result.download.return_value = [Path("/tmp/test.osm.pbf")]
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.osm.pbf"

(S108)